### PR TITLE
Improve PWM test app

### PR DESCRIPTION
**Pull request by hs0225**
- Add variable duty-cycle test
 - Separate sync and async test
 - Tested on rpi-linux, nuttx-stm32f4discovery and tizenrt-artik053

IoT.js-DCO-1.0-Signed-off-by: Hosung Kim hs852.kim@samsung.com

### CODE REVIEW

File: test/run_pass/systemio_common.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+var assert = require('assert');
+
+var pin = {};
+
+if (process.platform === 'linux') {
+  pin.pwm1 = 0;
+} else if (process.platform === 'nuttx') {
+  var stm32_pin = require('stm32f4dis').pin;
+  pin.pwm1 = stm32_pin.PWM1.CH1_1;
+} else if (process.platform === 'tizenrt') {
```
Comment (glistening) [inline test/run_pass/systemio_common.js]: @hs0225 What about mergin `tizenrt` and `linux`?

Comment (hs0225) [reply test/run_pass/systemio_common.js]: I will add pins of other modules here. So `tizenrt` and `linux` must be separated.

File: test/run_pass/test_pwm_async.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var assert = require('assert');
+var Pwm = require('pwm');
+var pin = require('systemio_common').pin;
```
Comment (daeyeon) [inline test/run_pass/test_pwm_async.js]: `tools/systemio_common`

Comment (hs0225) [reply test/run_pass/test_pwm_async.js]: I fixed it.

**Comment by glistening** [comment]
@hs0225 I expected 'systemio_common.js` contains the common code of system i/o. But the file mainly contains configuration for each platform, there is no common code. I would choose other name for the file like `systemio_config.js`.

**Comment by yichoi** [comment]
LGTM

**Comment by hs0225** [comment]
@glistening I'm going to put additional common features in the module to build the test framework.

**Pull request merged**